workflow-runtime/src/commonMain/kotlin/com/squareup/workflow1/RenderWorkflow.kt

```diff
@@ -189,15 +190,23 @@ public fun <PropsT, OutputT, RenderingT> renderWorkflowIn(
       var nextRenderAndSnapshot: RenderingAndSnapshot<RenderingT> = runner.nextRendering()
 
       if (runtimeConfig.contains(CONFLATE_STALE_RENDERINGS)) {
-        while (isActive && actionResult is ActionApplied<*> && actionResult.output == null) {
+        conflate@ while (isActive && actionResult is ActionApplied<*> && actionResult.output == null) {
+          // We start by yielding, because if we are on an Unconfined dispatcher, we want to give
+          // other signals (like Workers listening to the same result) a chance to get dispatched
+          // and queue their actions.
+          yield()
           // We may have more actions we can process, this rendering could be stale.
           actionResult = runner.processAction(waitForAnAction = false)
 
           // If no actions processed, then no new rendering needed. Pass on to UI.
-          if (actionResult == ActionsExhausted) break
+          if (actionResult == ActionsExhausted) break@conflate
 
           // Skip rendering if we had unchanged state, keep draining actions.
-          if (shouldShortCircuitForUnchangedState(actionResult)) continue
+          if (shouldShortCircuitForUnchangedState(actionResult)) {
+            // Emit the Output
```

Comment:
Nit: This comment is pretty redundant, "emit" is just another word for "send" and that's literally the function name.